app/src/main/java/org/fossasia/openevent/general/search/SearchLocationFragment.kt

```diff
@@ -31,22 +31,26 @@ class SearchLocationFragment : Fragment() {
     private val geoLocationViewModel by viewModel<GeoLocationViewModel>()
     private val safeArgs: SearchLocationFragmentArgs by navArgs()
 
-    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
+    private val AUTOCOMPLETE_FRAG_TAG = "AutoComplete_Frag"
+
+    override fun onCreateView(
+        inflater: LayoutInflater,
+        container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
```

Comment:
Unnecessary changes.

Reply:
Umm . This is for better view of parameters used.  Still don't want . I would revert it